deployment/docker/Dockerfile

```diff
@@ -11,9 +11,9 @@
 # __date__ = '13/06/2023'
 # __copyright__ = ('Copyright 2023, Unicef')
 
-FROM ubuntu:20.04 AS prod
```

Comment:
Also please don't include this to commit
I will update Dockerfile to split between arm and non arm